ui/lib/core/addon/components/kv-object-editor.js

```diff
@@ -86,6 +90,9 @@ export default class KvObjectEditor extends Component {
   }
   @action
   validateKey(rowIndex, event) {
+    if (this.args.isSingleRow) {
```

Comment:
I don't think validation should be reliant on `isSingleRow`, I think another arg should be passed that's called `allowWhiteSpace` or something